pkg/types/event.go

```diff
@@ -50,3 +51,12 @@ type EventWorkerLifecycleSchema struct {
 	WorkerID string `json:"worker_id"`
 	Status   string `json:"status"`
 }
+
+var EventAbstractionTriggerSchemaVersion = "1.0"
+
+type EventAbstractionTriggeredSchema struct {
```

Comment:
Shouldn't this have an event_type field or something? this feels ambiguous

Reply:
The stubtype includes the abstraction and event type